### PR TITLE
updated imagekit and modified readme

### PR DESCRIPTION
**Pull request by nick2496**
Hi,
-updated dependencies on both package.json and yarn.lock with latest ^versions - both are in sync
-updated README.md to reflect the accurate border as b

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -12229,11 +12229,6 @@ vue-template-es2015-compiler@^1.6.0, vue-template-es2015-compiler@^1.9.0:
   resolved "https://registry.yarnpkg.com/vue-template-es2015-compiler/-/vue-template-es2015-compiler-1.9.1.tgz#1ee3bc9a16ecbf5118be334bb15f9c46f82f5825"
   integrity sha512-4gDntzrifFnCEvyoO8PqyJDmguXgVPxKiIxrBKjIowvL9l+N66196+72XVYR8BBf1Uv1Fgt3bGevJ+sEmxfZzw==
 
-vue@^2.6.12:
```
Comment (imagekitio) [inline yarn.lock:12232]: Add `vue` as peer dependency so that it is not removed from `yarn.lock`

**Review by imagekitio — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (nick2496) [reply yarn.lock:12232]: ok

Comment (nick2496) [reply yarn.lock]: Hi, 
added vue as peer dependency

**Review by imagekitio — APPROVED** [review]

**Comment by imagekitio** [comment]
Awesome. Thanks for contributing!

**Pull request merged**